scripts/gqm_gen/README.md

```diff
@@ -2,11 +2,53 @@
 
 Creates a Mermaid diagram from Goal, Question, Metrics (GQM) Markdown files.
 
+The instructions below describe how to locally run 
+the script that builds the mermaid diagram. 
+A GitHub Actions will re-run this script at built time
+and regenerate the diagram, but it is good to run 
+locally to make sure it will build correctly.
+
+Be sure to also read the instructions in the 
+`measuring/use_gqm.md` file path.
+
+## Installation
+
+### Install mermaid CLI globally
+
+From this folder, follow instructions on the 
+[mermaid.js README](https://github.com/mermaid-js/mermaid-cli) to run in the terminal: 
+```
+npm install -g @mermaid-js/mermaid-cli
+```
+
+Install project specific node.js based on the package.json in this folder: 
+```
+npm install
+```
+
 ## Usage
 
 Builds the mermaid diagram and outputs it to stdout.
 
 ```bash
-npm install
-npm run --silent start
+npm run --silent start > gqm.mmd && mmdc --input gqm.md --output gqm.png
 ```
+
+Note that the file ending `.mmd` is not a typo. It is mermaid markdown file.
+
+This should generate a file called `gqm.png` or `gqm-1.png` in the same folder as this file.
+
+You will also end up with BOTH a `gqm.md` and a `gqm.mmd` file as the later is created upon running the script.
```

Comment:
No mmd file should be generated after adjusting the recommended command above.

<img width="936" alt="image" src="https://github.com/InnerSourceCommons/managing-innersource-projects/assets/776901/1c4a5d34-05ee-463a-8266-ff565d95c025">

Reply:
changed as suggested